parachain/primitives/core/src/assets.rs

```diff
@@ -1,20 +1,19 @@
 use frame_support::dispatch::DispatchResult;
 
-use codec::{Encode, Decode};
```

Comment:
Lets please leave major linting/whitespace changes for another PR.

Reply:
Sure, I will just remove the use of rustfmt as mentioned above.